src/utils/aspect-ratio-support.tsx

```diff
@@ -26,3 +27,92 @@ export const AspectRatioSupportProvider: React.FC<Props> = ({children}) => {
 };
 
 export const useSupportsAspectRatio = (): boolean => React.useContext(AspectRatioSupport);
+
+const useAspectRatioStyles = createUseStyles(() => ({
+    container: {
+        '@supports (aspect-ratio: 1 / 1)': {
+            aspectRatio: ({aspectRatio}) => aspectRatio ?? 'unset',
+        },
+        '$wrapper &': {
+            position: ({aspectRatio}) => (aspectRatio ? 'absolute' : 'static'),
+            width: '100%',
+            height: '100%',
+            top: 0,
```

Comment:
🤔 maybe you can get rid off `top` and `left` in case `aspectRatio` is not defined?